src/cli/repl.rs

```diff
@@ -535,6 +532,16 @@ impl Repl<F> {
             editor.load_history(history_path)?;
         }
 
+        let panic_on_repl_error = std::env::var("LURK_PANIC_ON_REPL_ERROR") == Ok("true".into());
```

Comment:
This [will return an error if the env var isn't set](https://doc.rust-lang.org/std/env/fn.var.html), and you probably want a panic to be the safe default.

One way to fix this is to refactor to a negative :

`let recover_repl_errors = std::env::var("LURK_PANIC_ON_REPL_ERROR") == Ok("false".to_owned())`

Reply:
No, panic is the exception for when we use the REPL on our test suites:
```lisp
λ LURK_PANIC_ON_REPL_ERROR=true cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.86s
     Running `target/debug/lurk`
commit: 2023-11-10 6df3c723b9078ef48b417f39d261f961225b0700
Lurk REPL welcomes you.
user> !(load "this_file_doesnt_exist")
thread 'main' panicked at src/cli/repl.rs:539:17:
!Error: Meta command failed with No such file or directory (os error 2)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

For the regular use case, we just want the REPL to print whatever went wrong:
```lisp
λ cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.15s
     Running `target/debug/lurk`
commit: 2023-11-10 6df3c723b9078ef48b417f39d261f961225b0700
Lurk REPL welcomes you.
user> !(load "this_file_doesnt_exist")
!Error: Meta command failed with No such file or directory (os error 2)
user> 
```

Reply:
Actually, wait. This can be simpler